--- a/qt-everywhere-opensource-src-4.8.4/src/corelib/plugin/qlibrary.cpp	2013-04-25 12:14:28.883972119 +0200
+++ b/qt-everywhere-opensource-src-4.8.4/src/corelib/plugin/qlibrary.cpp	2013-04-25 12:00:37.299973839 +0200
@@ -722,7 +722,7 @@
         qt_version = reg.at(0).toUInt(0, 16);
         debug = bool(reg.at(1).toInt());
         key = reg.at(2).toLatin1();
-        success = qt_version != 0;
+        success = qt_version == QT_VERSION;
     } else {
 #if defined(Q_OS_UNIX) && !defined(Q_OS_MAC) && !defined(Q_OS_SYMBIAN)
         if (!pHnd) {
