--- a/qt-everywhere-opensource-src-4.8.5/src/corelib/plugin/qlibrary.cpp	2013-04-25 12:14:28.883972119 +0200
+++ b/qt-everywhere-opensource-src-4.8.5/src/corelib/plugin/qlibrary.cpp	2013-04-25 12:00:37.299973839 +0200
@@ -722,7 +722,7 @@
         qt_version = reg.at(0).toUInt(0, 16);
         debug = bool(reg.at(1).toInt());
         key = reg.at(2).toLatin1();
-        success = qt_version != 0;
+        success = qt_version == QT_VERSION;
     } else {
 #if defined(Q_OS_UNIX) && !defined(Q_OS_MAC) && !defined(Q_OS_SYMBIAN)
         if (!pHnd) {
@@ -834,7 +834,7 @@
 
     pluginState = IsNotAPlugin; // be pessimistic
 
-    if ((qt_version & 0x00ff00) > (QT_VERSION & 0x00ff00) || (qt_version & 0xff0000) != (QT_VERSION & 0xff0000)) {
+    if (qt_version != QT_VERSION) {
         if (qt_debug_component()) {
             qWarning("In %s:\n"
                  "  Plugin uses incompatible Qt library (%d.%d.%d) [%s]",
--- a/qt-everywhere-opensource-src-4.8.5/src/3rdparty/webkit/Source/common.pri	2013-06-07 01:16:58.000000000 -0400
+++ b/qt-everywhere-opensource-src-4.8.5/src/3rdparty/webkit/Source/common.pri	2013-07-16 16:02:18.458414176 -0400
@@ -6,7 +6,6 @@
 linux-g++ {
 isEmpty($$(SBOX_DPKG_INST_ARCH)):exists(/usr/bin/ld.gold) {
     message(Using gold linker)
-    QMAKE_LFLAGS+=-fuse-ld=gold
 }
 }
 
